server/src/core/@generated/person/person-create-many-company.input.ts

```diff
@@ -20,11 +20,11 @@ export class PersonCreateManyCompanyInput {
     lastName!: string;
 
     @Field(() => String, {nullable:false})
-    @Validator.IsEmail()
+    @Validator.IsString()
```

Comment:
Could `@IsOptional()` or another `class-validator` decorator be a more secure alternative ?

Reply:
This is to be discussed IMO. In the current design, a non-set email is the "" empty string. IsOptional() + IsEmail() means `undefined | email`. What we would like is `"" | email`
These validations were added too quickly!